### DISCUSSION TITLE
Mountain Duck

### DISCUSSION THREAD

**Discussion by Crisveras** [iterate-ch/cyberduck#15996] [Q&A]
I'm having a problem with the Mountain Duck software with a registered user, I have a Windows machine with multi-users, I need all users to access mountain duck, but every time I try to register more than 2 users on the machine, it gives an invalid key and the key is valid, regardless of whether it is Windows 7 or 10, and this problem happened with 5 different machines and it does not work. I would like help to solve this problem.

Estou tendo problema com a software Mountain Duck com cadastrado de usuário, tenho uma máquina Windows com multiusuários, necessito que todos os usuários acessem o mountain duck, mas toda vez que eu tento cadastrar mais de 2 usuários na máquina, dar chave invalida sendo que a chave está válida, independente que seja Windows 7 ou 10, e esse problema aconteceu com 5 máquinas diferentes e não funciona. Gostaria de ajuda para solucionar esse problema.

**Comment by tatze96** [comment]
The "Not a valid registration key" error can only occur in three cases.

1. You own a license for a previous version that isn't valid for the currently installed Mountain Duck version.
2. Your license file doesn't have the correct file extension (.mountainducklicense).
3. The license file got damaged in another way.

The first case doesn't apply to your license based on your description.

Please check if the license has the correct file extension. If it has the correct file extension but you still get the "Not a valid registration key" message, the license file likely got altered during the download from the email attachments.

If this is the case, try redownloading it through a different email application or web browser and applying the license again.

You can also copy the license file into the [application support folder](https://docs.cyberduck.io/mountainduck/support/#application-support-folder) if double-clicking on it doesn't apply the license correctly.
